DESCRIPTION

```diff
@@ -1,7 +1,6 @@
 Type: Package
 Package: ooklaOpenDataR
-Title: Speedtest by Ookla Global Fixed and Mobile Network
-    Performance Map Tiles
+Title: Work with 'Speedtest by Ookla' Global Internet Performance Data
```

Comment:
The first 2 words might be considered superfluous by the CRAN maintainers?

```suggestion
Title: 'Speedtest by Ookla' Global Internet Performance Data
```

Reply:
I definitely don't like "work with", but my worry is that the package itself doesn't contain data, rather it helps retrieve and filter the data

Reply:
Gotcha 👍